packages/sdk/src/lib/dataProtectorSharing/consumeProtectedData.ts

```diff
@@ -39,9 +39,10 @@ export const consumeProtectedData = async ({
   iexec = throwIfMissing(),
   sharingContractAddress = throwIfMissing(),
   protectedData,
-  maxPrice = DEFAULT_MAX_PRICE,
   app,
+  path,
   workerpool,
+  maxPrice = DEFAULT_MAX_PRICE,
```

Comment:
Move this param later in the list of params, to be consistent with ordering in TypeScript type.